goth/runner/process.py

```diff
@@ -34,20 +35,20 @@ async def run_command(
     if log_prefix is None:
         log_prefix = f"[{args[0]}] "
 
-    p = subprocess.Popen(
-        args=args, env=env, stdout=subprocess.PIPE, stderr=subprocess.STDOUT
-    )
+    async def _run_command():
 
-    async def _read_output():
-        for line in p.stdout:
-            logger.debug("%s%s", log_prefix, line.decode("utf-8").rstrip())
+        proc = await asyncio.subprocess.create_subprocess_exec(
+            *args, env=env, stdout=subprocess.PIPE, stderr=subprocess.STDOUT
+        )
 
```

Comment:
Using `asyncio.subprocess` instead of `subprocess` will allow us to read the output from the command without blocking all other `asyncio` tasks until the command finishes.